docs/configuration.asciidoc

```diff
@@ -551,7 +551,18 @@ A capture location stack trace is never generated for uncaught exceptions.
 * *Default:* `true`
 * *Env:* `ELASTIC_APM_CAPTURE_SPAN_STACK_TRACES`
 
-Set this option to `false` to disable capture of stack traces for measured spans during instrumentation.
+Set this option to `false` to disable capture of stack traces.
+
+When set to `true` the agent will capture stack traces for spans whose duration falls within the <<span-frames-min-duration, `spanFramesMinDuration`>> threshold.
+
+[[span-frames-min-duration]]
+==== `spanFramesMinDuration`
+
+* *Type:* String
+* *Default:* `10ms`
+* *Env:* `ELASTIC_APM_CAPTURE_SPAN_STACK_TRACES_THRESHOLD`
```

Comment:
Should this instead be `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` ?

Reply:
yup, good eye.  ✅